From 4877f9dcf8dab27f05917c8a7c8f0971ef54fa10 Mon Sep 17 00:00:00 2001
From: Andreas Robinson <andr345@gmail.com>
Date: Sat, 3 Oct 2009 20:18:35 +0200
Subject: [PATCH 08/19] modprobe: cleanup indentation change in conf parser

Follow up to the previous commit.

Signed-off-by: Andreas Robinson <andr345@gmail.com>
(cherry picked from commit bc334ddccd70b30f79983d1dff6eef2e652d054a)
---
 modprobe.c | 44 +++++++++++++++++++++-----------------------
 1 file changed, 21 insertions(+), 23 deletions(-)

diff --git a/modprobe.c b/modprobe.c
index 43730f0..1d4189a 100644
--- a/modprobe.c
+++ b/modprobe.c
@@ -943,7 +943,6 @@ static int parse_config_file(const char *filename,
 		if (streq(cmd, "alias")) {
 			char *wildcard = strsep_skipspace(&ptr, "\t ");
 			char *realname = strsep_skipspace(&ptr, "\t ");
-
 			if (!wildcard || !realname)
 				goto syntax_error;
 			if (fnmatch(underscores(wildcard),name,0) == 0)
@@ -951,38 +950,37 @@ static int parse_config_file(const char *filename,
 		} else if (streq(cmd, "include")) {
 			struct module_alias *newalias = NULL;
 			char *newfilename;
-
 			newfilename = strsep_skipspace(&ptr, "\t ");
 			if (!newfilename)
 				goto syntax_error;
 
-				warn("\"include %s\" is deprecated, "
-				     "please use /etc/modprobe.d\n", newfilename);
-				if (strstarts(newfilename, "/etc/modprobe.d")) {
-					warn("\"include /etc/modprobe.d\" is "
-					     "the default, ignored\n");
-				} else {
-					if (!parse_config_scan(newfilename, name,
-							      dump_only, removing,
-							      options, commands, &newalias,
-							      blacklist))
-						warn("Failed to open included"
-						      " config file %s: %s\n",
-						      newfilename, strerror(errno));
-				}
-				/* Files included override aliases,
-				   etc that was already set ... */
-				if (newalias)
-					*aliases = newalias;
+			warn("\"include %s\" is deprecated, "
+			     "please use /etc/modprobe.d\n", newfilename);
+			if (strstarts(newfilename, "/etc/modprobe.d")) {
+				warn("\"include /etc/modprobe.d\" is "
+				     "the default, ignored\n");
+			} else {
+				if (!parse_config_scan(newfilename, name,
+						      dump_only, removing,
+						      options, commands, &newalias,
+						      blacklist))
+					warn("Failed to open included"
+					      " config file %s: %s\n",
+					      newfilename, strerror(errno));
+			}
+			/* Files included override aliases,
+			   etc that was already set ... */
+			if (newalias)
+				*aliases = newalias;
 
 		} else if (streq(cmd, "options")) {
 			modname = strsep_skipspace(&ptr, "\t ");
 			if (!modname || !ptr)
 				goto syntax_error;
 
-				ptr += strspn(ptr, "\t ");
-				*options = add_options(underscores(modname),
-						       ptr, *options);
+			ptr += strspn(ptr, "\t ");
+			*options = add_options(underscores(modname),
+					       ptr, *options);
 
 		} else if (streq(cmd, "install")) {
 			modname = strsep_skipspace(&ptr, "\t ");
-- 
1.9.0

